sky/clouds/cloud.py

```diff
@@ -297,5 +297,16 @@ def need_cleanup_after_preemption(self,
         del resource
         return False
 
+    @classmethod
+    def support(cls, requested_features: List[str]) -> bool:
```

Comment:
nit: typically, `supports()` / `exists()` for bool

Reply:
I changed `support` -> `supports`.